src/main/java/de/tum/in/www1/artemis/service/connectors/bitbucket/BambooBuildPlanUpdateService.java

```diff
@@ -0,0 +1,184 @@
+package de.tum.in.www1.artemis.service.connectors.bitbucket;
+
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import javax.annotation.Nullable;
+import javax.validation.constraints.NotNull;
+
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Profile;
+import org.springframework.http.*;
+import org.springframework.stereotype.Service;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.util.UriComponentsBuilder;
+
+import de.tum.in.www1.artemis.exception.BambooException;
+import de.tum.in.www1.artemis.service.connectors.bamboo.BambooInternalUrlService;
+import de.tum.in.www1.artemis.service.connectors.bamboo.dto.BambooRepositoryDTO;
+import de.tum.in.www1.artemis.service.connectors.ci.ContinuousIntegrationUpdateService;
+
+@Service
+// Only activate this service bean, if both Bamboo and Bitbucket are activated (@Profile({"bitbucket","bamboo"}) would activate
+// this if any profile is active (OR). We want both (AND)
+@Profile("bamboo")
```

Comment:
Currently, this is only checking for bamboo. Where are we checking for bitbucket?